### PR TITLE
chore: update CODEOWNERS for bridge related code

### PR DESCRIPTION
**Pull request by infiniteflower**
## **Description**

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/29029?quickstart=1)

This PR updates `CODEOWNERS` for all Bridge related code.

## **Related issues**

n/a

## **Manual testing steps**

n/a
## **Screenshots/Recordings**

n/a

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -96,10 +96,14 @@ ui/pages/settings/security-tab/profile-sync-toggle      @MetaMask/identity
...
 app/scripts/lib/snap-keyring         @MetaMask/accounts-engineers
 
-# Swaps team to own code for the swaps folder
+# Swaps-Bridge team to own code for the swaps folder
 ui/pages/swaps                                        @MetaMask/swaps-engineers
 app/scripts/controllers/swaps                         @MetaMask/swaps-engineers
 
+# Swaps-Bridge team to own code for the bridge folder
+**/bridge/**                                          @MetaMask/swaps-engineers
+**/bridge-status/**                                    @MetaMask/swaps-engineers
```
Comment (micaelae) [inline .github/CODEOWNERS:105]: Nit: remove 1 space before `@`

Comment (infiniteflower) [reply .github/CODEOWNERS]: Done in [7aca15d](https://github.com/MetaMask/metamask-extension/pull/29029/commits/7aca15d8a8fbadb2676244f9fa8cdfb277e28780)

**Review by micaelae — APPROVED** [review]

**Review by dbrans — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -96,10 +96,14 @@ ui/pages/settings/security-tab/profile-sync-toggle      @MetaMask/identity
 
 app/scripts/lib/snap-keyring         @MetaMask/accounts-engineers
 
-# Swaps team to own code for the swaps folder
+# Swaps-Bridge team to own code for the swaps folder
 ui/pages/swaps                                        @MetaMask/swaps-engineers
 app/scripts/controllers/swaps                         @MetaMask/swaps-engineers
 
+# Swaps-Bridge team to own code for the bridge folder
+**/bridge/**                                          @MetaMask/swaps-engineers
```
Comment (jclancy93) [inline .github/CODEOWNERS:104]: I see we have types, like bridge-status.ts that lives outside of this structure. For files like these, should we:

* colocate them in this or a follow on PR?
* Add specific entries for those files here?

Not blocking, just wanted to call out

**Review by ejwessel — APPROVED** [review]

**Review by davidmurdoch — APPROVED** [review]

**Review by desi — APPROVED** [review]
LGTM

File: .github/CODEOWNERS (hunk 2 above)
Comment (infiniteflower) [reply .github/CODEOWNERS:104]: We can make a new PR to add additional files

**Pull request merged**